WifiART/src/userinterface/ArtMvcController.java

```diff
@@ -27,20 +30,42 @@
      *
      * */
     public ArtMvcController(ArtMvcView fv, ArtMvcModel fm) {
-        this.fm = fm;
-        this.fv = fv;
+        this.model = fm;
+        this.view = fv;
 
         // Initialize GUI by copying state from model
-        this.fv.setViewState(fm.getState());
-        this.fm.setCallback(this);
+        this.view.setViewState(fm.getState());
+        this.model.setCallback(this);
 
         // TODO(dmevans) Add the listeners to the View here
```

Comment:
Has this ToDo been completed?

Reply:
This is a first draft, I included it because I wanted to demonstrate how I was planning on doing the listeners. I'll delete the TODO when I feel that this section is done, possibly in my revision.